src/store/mutations/mutations-model.ts

```diff
@@ -5,3 +5,11 @@ export const mutateName = (state: any, name: string) => {
 export const mutateResource = (state: any, resource: object) => {
   state.resourceModel = resource
 }
+
+export const mutateKeycloakRoles = (state: any, keyCloakRoles: Array<string>) => {
```

Comment:
Is the intent for 'keyCloakRoles' and 'authRoles' below to use the interfaces you declared above?

Reply:
Not in these instances. The mutators refer to specific values and therefor just need the type of that specific property, whereas the interfaces were more for defining the model as a whole.

Reply:
Ah, I see!